syntax = "proto3";

package userLocation;

service UserLocationService {
  rpc SendLocation (UserRequest) returns (LocationResponse) {}
}

message UserRequest {
  string userId = 1;
}

message LocationResponse {
  string userId = 1;
  float latitude = 2;
  float longitude = 3;
}
